build.sbt

```diff
@@ -23,4 +21,48 @@ initialCommands in console :=
 
 libraryDependencies += "com.twitter" %% "util-jvm" % "6.23.0" % "provided"
 
-libraryDependencies += "org.scalatest" %% "scalatest" % "2.2.1" % "test"
\ No newline at end of file
+libraryDependencies += "org.scalatest" %% "scalatest" % "2.2.1" % "test"
+
+// Your username to login to Databricks Cloud
+dbcUsername := sys.env.getOrElse("DBC_USERNAME", sys.error("Please set DBC_USERNAME"))
+
+// Your password (Can be set as an environment variable)
+dbcPassword := sys.env.getOrElse("DBC_PASSWORD", sys.error("Please set DBC_PASSWORD"))
+
+// The URL to the Databricks Cloud DB Api. Don't forget to set the port number to 34563!
+dbcApiUrl := sys.env.getOrElse ("DBC_URL", sys.error("Please set DBC_URL"))
+
+// Add any clusters that you would like to deploy your work to. e.g. "My Cluster"
+// or run dbcExecuteCommand
+dbcClusters += sys.env.getOrElse("DBC_USERNAME", sys.error("Please set DBC_USERNAME"))
+
+dbcLibraryPath := s"/Users/${sys.env.getOrElse("DBC_USERNAME", sys.error("Please set DBC_USERNAME"))}/lib"
+
+import ReleaseTransformations._
+
+/** Push to the team directory instead of the user's homedir for releases. */
+lazy val setupDbcRelease = ReleaseStep(
+  action = { st: State =>
+    val extracted = Project.extract(st)
+    val newSettings = extracted.structure.allProjectRefs.map { ref =>
+      dbcLibraryPath in ref := "/databricks/spark/sql/lib"
+    }
+
+    reapply(newSettings, st)
+  }
+)
+
+// Add publishing to spark packages as another step.
+releaseProcess := Seq[ReleaseStep](
+  checkSnapshotDependencies,
+  inquireVersions,
+  runTest,
+  setReleaseVersion,
+  commitReleaseVersion,
+  tagRelease,
+  setupDbcRelease,
+  releaseStepTask(dbcDeploy),
```

Comment:
Maybe use `dbcUpload` instead of `dbcDeploy` because `dbcDeploy` will try to attach to a cluster. If the cluster does not exist, the release process will fail.